src/services/mock/graphql.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const data = require('./json/index')
+
+/**
+ * Mapping between a query and the returned data. The returned data
+ * can be generated programmatically, or with GraphiQL, or another GraphQL
+ * command line.
+ *
+ * @type {{GscanSubscriptionQuery: {workflows: {}}, play: {data: {play: {result: [{response: (boolean|string)[], id: string}], __typename: string}}}, resume: {data: {resume: {result: [{response: (boolean|string)[], id: string}], __typename: string}}}, stop: {data: {stop: {result: [{response: (boolean|string)[], id: string}], __typename: string}}}, IntrospectionQuery: {errors: {}, data: {__schema: {queryType: {kind: string, name: string, description: string, fields: {}, inputFields: *, interfaces: {}, enumValues: *, possibleTypes: *, __typename: string}, mutationType: {kind: string, name: string, description: *, fields: {}, inputFields: *, interfaces: {}, enumValues: *, possibleTypes: *, __typename: string}, subscriptionType: {name: string, __typename: string}, types: {}, directives: {}}}}, OnWorkflowTreeDeltasData: {deltas: {added: {workflow: {id: string, name: string, status: string, owner: string, host: string, port: number, stateTotals: {waiting: number, expired: number, preparing: number, 'submit-failed': number, submitted: number, running: number, failed: number, succeeded: number}, latestStateTasks: {running: {}}, cyclePoints: {}, taskProxies: {}, familyProxies: {}}}}}, WorkflowsTableQuery: {workflows: {}}, pause: {data: {pause: {result: [{response: (boolean|string)[], id: string}], __typename: string}}}, DashboardSubscriptionQuery: {workflows: {}}}}
+ */
+const DATA_MAPPING = {
+  IntrospectionQuery: data.IntrospectionQuery,
+  OnWorkflowTreeDeltasData: {
+    deltas: {
+      added: {
+        workflow: data.OnWorkflowTreeDeltasData.workflows[0]
+      }
+    }
+  },
+  DashboardSubscriptionQuery: data.DashboardSubscriptionQuery,
+  GscanSubscriptionQuery: data.GscanSubscriptionQuery,
+  WorkflowsTableQuery: data.WorkflowsTableQuery
```

Comment:
The key/left is what should be in the client GraphQL query (why I changed `queries.js`). Right is the JSON object returned.